labels/en-US.json

```diff
@@ -553,7 +553,9 @@
   "returnReceiptItemUPCColumn": "UPC",
   "returnReceiptItemQtyColumn": "Qty",
   "returnReceiptItemStatusColumn": "Status",
-  "returnReceiptItemTotalColumn": "Total",
+  "returnReceiptItemTotalColumn": "Item Total",
+  "returnReceiptReductionAmountColumn": "Return Reduction Amount",
+  "returnReceiptUnitPriceColumn": "Unit Price",
```

Comment:
seeing some discrepancies in labels files like "returnReceiptUnitPriceColumn" key which you have added into en-US.json is not present in de-DE.josn. Make sure your newly added keys are present in both files.

Reply:
ditto

Reply:
Done